sprints/frontend/src/components/Table.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+const largeColumn = {width: '40%'};
+const midColumn = {width: '30%'};
+const smallColumn = {width: '10%'};
+
+
+const Table = ({list}) =>
+    <div className="table">
```

Comment:
Nit: it's usually best to use an actual HTML `<table className="table">` for tabular data; it has better accessibility properties and it allows users to copy-and-paste the data into a spreadsheet application and then use make calculations using it.